msnc/model.py

```diff
@@ -99,27 +99,22 @@ def run_training(
 
             if development_set is not None:
                 dev_accuracy = self.run_evaluation(development_set)
-                log_line = self._set_log_line(dev_accuracy, epoch)
-                logger.info(log_line)
+                self._write_log(dev_accuracy, epoch)
 
                 if not self.is_best(dev_accuracy):
                     continue
 
                 # When the new model outperforms the previous ones
-                log_line = self._set_log_line(dev_accuracy, epoch, '[new best]')  # NOQA
+                self._write_log(dev_accuracy, epoch, '[new best]')
 
                 # Save epoch and performance information
                 self._best_epoch = epoch
                 self._best_dev_accuracy = dev_accuracy
 
                 if save_best_model:
                     self._save('best.model')
-                    logger.debug('Update best model')
 
-                logging.info(log_line)
```

Comment:
Are there any differences between `logger.info(log_line)` and `logging.info(log_line)`? 🤔

Reply:
logging.info shuld be replaced with logger.info
I'm sorry. 🙇 :bow